### ISSUE TITLE
Add version numbers

### ISSUE THREAD

**Issue by johnanvik** [johnanvik/program-wars#87]
It is hard to know which version is running on the server. We should find a way to annotate or indicate which version is running. This will help with future bug reports.

**Comment by vincentcote87** [comment]
is the version number something we change in the code every time a new pull request is made or is it usually something that updates dynamically? Currently the version number is displayed in the credits modal, I can go in and update it as of v.1.1 and update on each future milestones?

**Comment by johnanvik** [comment]
Ideally, I'd like to figure out a way to do this dynamically so it is updated when there is a deployment, but I'm not sure how to do that. Issue #85 will need to get fixed before we start figuring this one out.

**Comment by vincentcote87** [comment]
Sounds good, for now ill update the credits for #79 and hard code v.1.1.
